packages/shared-ui/src/i18n/en.json

```diff
@@ -278,6 +278,10 @@
     "identifier_not_allowed": "Couldn't find your account",
     "login_identifier_unknown_user": "Couldn't find your account.",
     "invalid_conditional_ui_login": "The provided passkey is no longer valid. Please enter your identifier (e.g. email) manually.",
+    "no_fallback_available": {
```

Comment:
We don't need to distinguish between withCustomerSupport and noCustomerSupport here I think. IMO our error on top of the component are now used in two situations:
- something unexpected has happened => we show a static generic error message (here we can show the customerSupport email)
- something expected has happened (e.g. no_fallback_available) and we want to show it on top of the component => this error message does not include customerSupport email